tools/stimulus/types-RC.yaml

```
@@ -39,7 +39,9 @@ INTEGER:
         OUT: '&%C%'
         INOUT: '&%C%'
     INCONV:
-        IN: '%C%=(igraph_integer_t) REAL(%I%)[0];'
+        IN: |-
+            IGRAPH_R_CHECK_INT(%I%);
+            %C% = (igraph_integer_t) REAL(%I%)[0];
```

For the future: we could do something like 

```suggestion
            %C% = IGRAPH_R_CHECK_INT(%I%);
```

with a suitable implementation of `IGRAPH_R_CHECK_INT()` . It's good as is for now.

---

Same for the other check functions.